src/flixel/physics/FlxQuadTree.as

```diff
@@ -262,16 +275,13 @@ package flixel.physics
 			if(_headA != null)
 				_headA.destroy();
 			_headA = null;
-			if(_tailA != null)
-				_tailA.destroy();
```

Comment:
`_headA` and `_tailA` point to the same object. We can't call `destroy()` on it twice. See the modified implementation in `FlxList` for this commit.